src/LiveComponent/src/LiveComponentHydrator.php

```diff
@@ -354,6 +354,10 @@ private function dehydrateValue(mixed $value, LivePropMetadata $propMetadata, ob
         }
 
         if ($propMetadata->useSerializerForHydration()) {
+            if (!interface_exists(NormalizerInterface::class)) {
+                throw new \LogicException(sprintf('The LiveProp "%s" on component "%s" is using the serializer to hydrate, but the Serializer component is not installed. Try running "composer require symfony/serializer".', $propMetadata->getName(), $parentObject::class));
```

Comment:
Let's mention the specific option name:

```suggestion
                throw new \LogicException(sprintf('The LiveProp "%s" on component "%s" has "useSerializerForHydration: true" but the Serializer component is not installed. Try running "composer require symfony/serializer".', $propMetadata->getName(), $parentObject::class));
```